### ISSUE TITLE
mAP won't go above 50 and avg loss stays around 1.4/1.5

### ISSUE THREAD

**Issue by roelofd** [pjreddie/darknet#1664]
I want to train Darknet tiny YOLO to recognize playing cards. 
Some info:
Dataset = 1000 images 
Classes amt = 7
mAP test set amt = 200

my tiny yolo cfg is as follows: 
`[net]
# Testing
#batch=64
#subdivisions=8
# Training
batch=64
subdivisions=8
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
flip=0

learning_rate=0.0001
burn_in=1000
max_batches = 14000
policy=steps
steps=11200,12600
scales=.1,.1`

and my filter values above the {yolo] layers is 36
I have not recalculated the anchors and the weight i used is "yolov3-tiny.conv.
I have done about 1500 iterations at this point and since 1000 iterations there is barely any chance in results.

Is there anybody that knows what the problem might be? 
Could this possibly my dataset?

**Comment by kingjam3s** [comment]
I had a problem with the mAP and the problem was the dataset.
but maybe you have a different problem...

**Comment by harsh52** [comment]
There may be several reasons for that, if you have done everything right and your test dataset size is sufficient enough then i will highly recommend you to look at this tutorial for fine tune your object detection.
https://github.com/jaskarannagi19/yolov3/blob/master/README.md#how-to-train-to-detect-your-custom-objects

**Comment by 2008hsk** [comment]
I have reached 54 mAP after 3500 iterations and applying different learning rates.
